docs/sources/variables/advanced-variable-format-options.md

```diff
@@ -0,0 +1,135 @@
++++
+title = "Advanced variable format options"
+keywords = ["grafana", "templating", "documentation", "guide", "template", "variable"]
+type = "docs"
+[menu.docs]
+name = "advanced-variable-format-options"
+parent = "variables"
+weight = 300
++++
+
+# Advanced variable format options
+
+The formatting of the variable interpolation depends on the data source, but there are some situations where you might want to change the default formatting. 
+
+For example, the default for the MySql data source is to join multiple values as comma-separated with quotes: `'server01','server02'`. In some cases, you might want to have a comma-separated string without quotes: `server01,server02`. You can make that happen with advanced variable formatting options listed below.
+
+## General syntax
+
+Syntax: `${var_name:option}`
+
+Test the formatting options on the [Grafana Play site](https://play.grafana.org/d/cJtIfcWiz/template-variable-formatting-options?orgId=1).
+
+If any invalid formatting option is specified, then `glob` is the default/fallback option.
+
+An alternative syntax (that might be deprecated in the future) is `[[var_name:option]]`.
```

Comment:
We should add that there is a `raw` option too that means do not do any formatting. You can see an example on https://play.grafana.org/d/cJtIfcWiz/template-variable-formatting-options?orgId=1